daemon/mgr/container.go

```diff
@@ -132,8 +132,36 @@ func (mgr *ContainerManager) Restore(ctx context.Context) error {
 }
 
 // Remove removes a container, it may be running or stopped and so on.
-// TODO
 func (mgr *ContainerManager) Remove(ctx context.Context, name string, option *ContainerRemoveOption) error {
+	c, err := mgr.container(name)
+	if err != nil {
+		return err
+	}
+	c.Lock()
+	defer c.Unlock()
+
+	if !c.IsStopped() && !c.IsCreated() && !option.Force {
+		return fmt.Errorf("container: %s is not stopped, can't remove it", c.ID())
```

Comment:
When removing a running container without flag force, I think pouchd would return an error `container: %s is not stopped, can't remove it`.
I suggest using `fmt.Errorf("container %s is not stopped, cannot remove it without flag force", c.ID())` to tell others to use `force` flag.